src/_locales/en/messages.json

```diff
@@ -0,0 +1,109 @@
+  },
+  "addCurrentSiteToMySitesContextMenu": {
+    "message": "Add $sld$ to My External Sites",
+    "placeholders": { "sld": { "content": "$1" } }
+  },
+  "changeDefaultLaunchContextMenu": {
+    "message": "Always use Firefox Private Browsing"
+  },
+  "notInstalledErrorHeading": {
+    "message": "Firefox is not installed"
+  },
+  "notInstalledErrorText": {
+    "message": "Firefox Launch requires Firefox to be installed on your computer."
+  },
+  "notInstalledErrorAction": {
+    "message": "Download Firefox"
+  },
+  "openWelcomePage": {
+    "message": "Extension Settings"
+  },
+  "welcomePageTitleFirefox": {
+    "message": "Launch in Another Browser"
+  },
+  "welcomePageSubtitleFirefox": {
+    "message": "Switch between browsers in one click."
+  },
+  "welcomePageDescriptionFirefox": {
+    "message": "Sometimes, you just want to use a different browser.\nThis extension helps you easily open sites in other browsers."
+  },
+  "welcomePageTitleChromium": {
+    "message": "Launch in Firefox"
+  },
+  "welcomePageSubtitleChromium": {
+    "message": "Switch from your current browser to Firefox in one click."
+  },
+  "welcomePageDescriptionChromium": {
+    "message": "Sometimes, you just want a more private browser. This extension helps you easily open sites in Firefox."
+  },
+  "welcomePageBrowserSelector": {
+    "message": "Select other browser"
+  },
+  "welcomePageAlwaysPrivateCheckbox": {
+    "message": "Always open sites in a Firefox Private window when switching"
+  },
+  "welcomePageTelemetryCheckbox": {
+    "message": "Allow Mozilla to collect information about how you use this extension so we can make it better. {LinkStart}Privacy Notice{LinkEnd}"
+  },
+  "welcomePageNoShortcutsFirefox": {
+    "message": "You don't have any shortcuts set up for this extension. Create them by going to {LinkStart}extension settings.{LinkEnd}"
+  },
+  "welcomePageNoShortcutsChromium": {
+    "message": "You don't have any shortcuts set up for this extension. Create them by going to {LinkStart}extension settings.{LinkEnd}"
+  },
+  "welcomePageNoShortcutTo": {
+    "message": "You don't have a shortcut set up for launching your current page in $browser$",
+    "placeholders": { "browser": { "content": "$1" } }
+  },
+  "welcomePageYesShortcutTo": {
+    "message": "$hotkey$ launches your current page in $browser$",
+    "placeholders": {
+      "hotkey": { "content": "$1" },
+      "browser": { "content": "$2" }
+    }
+  },
+  "welcomePageShortcutTitle": {
+    "message": "Shortcuts"
+  },
+  "welcomePageManageShortcutsFirefox": {
+    "message": "Find shortcuts for this extension by going to {LinkStart}extension settings.{LinkEnd}"
+  },
+  "welcomePageManageShortcutsChromium": {
+    "message": "Find shortcuts for this extension by going to {LinkStart}extension settings.{LinkEnd}"
+  },
+  "welcomePageTryFirefox": {
+    "message": "Want to Launch in Firefox? {LinkStart}Try our Chromium extension.{LinkEnd}"
```

Comment:
Why Launch uppercase? That should be lowercase at this point.

Reply:
The temp name had changed to Launch in Firefox and Launch in Another Browser. I'll change Firefox Launch references to these names, but I will change it again later when the name is decided on.